docs/getting-started/install-guide.md

```diff
@@ -76,7 +86,33 @@ The Windows Installer (`.msi` file) is located here: [https://github.com/decred/
 
     You can either double click it or run it from the Command Prompt. 
     
-3. The binaries for `dcrd`, `dcrwallet`, and `dcrctl` can then be found in the `%HOMEPATH%\decred\` directory (usually %HOMEPATH% is `C:\Users\username`).
+3. The binaries for `dcrd`, `dcrwallet`, and `dcrctl` can then be found in the `%HOMEPATH%\decred\` directory (usually %HOMEPATH% is `C:\Users\username`). Before the `dcrinstall` completes, you will be taken to the wallet creation prompt. Follow the steps within the [Wallet Creation Walkthrough](/getting-started/user-guides/dcrwallet-setup.md#wallet-creation-walkthrough) of the dcrwallet Setup guide to finish.
```

Comment:
A little inconsistent in your tense.  'The binaries for ... can NOW be found' or even just 'can be found'.